packages/contracts/contracts/voting/multisig/Multisig.sol

```diff
@@ -123,23 +124,27 @@ contract Multisig is PluginUUPSUpgradeable, ProposalUpgradeable, Addresslist {
     ) public initializer {
         __PluginUUPSUpgradeable_init(_dao);
 
-        // add member addresses to the address list
         _addAddresses(_members);
 
         _updateMultisigSettings(_multisigSettings);
+
+        emit MembersAdded({members: _members});
+
+        emit MultisigSettingsUpdated({
```

Comment:
why not emitting ``MultisigSettingsUpdated` as before inside _updateMultisigSettings ?

Reply:
It is cleaner to emit the events from the top-level, external functions calling the internal functions like we do for `_addAddresses`.
This way, `_updateMultisigSettings` could be used in another context, where we don't need to emit the event.

Reply:
You emit 2 times(duplicated). It should be emitting from _updateMultisigSettings. :)

Reply:
Any update about this comment? We shouldn't be emitting events twice

Reply:
We don't emit it twice. The event is emitted in `initialize` once and in 
```
function updateMultisigSettings(MultisigSettings calldata _multisigSettings)
        external
        auth(UPDATE_MULTISIG_SETTINGS_PERMISSION_ID)
```
every time we update.

Since `updateMultisigSettings` requires a permission, we cannot put it into `initialize`.
Giorgi suggests that I put it into `_updateMultisigSettings`.
I didn't put it in `_updateMultisigSettings` for consistency with `_addressesAdded` where we also don't emit `MembersAdded`.

Note, that we shouldn't put `emit MembersAdded({members: _members});` into `_addAddresses` for the reasons outlined in other comments.

Reply:
Here you go e34e995a5d59e05657d930c36d18ffde3ad15cb1